iac-extensions/docker/src/test/resources/checks/SecretsHandlingCheck/SecretsHandlingCheck_arg.dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM foo
+
+
+# Noncompliant@+1 {{Make sure that using ARG to handle a secret is safe here.}}
+ARG ACCESS_TOKEN=ThisIsSomethingThatShouldProbablyBeSecret
+#   ^^^^^^^^^^^^
+# Noncompliant@+1
+ARG TOKEN=AAAA
+# Noncompliant@+1
+ARG token=AAAA
+# Noncompliant@+1
+ARG ACCESS_TOKEN="hts://www.secrets.com"
+# Noncompliant@+1
+ARG MY_1_ACCESS_TOKEN=AAAA
+# Noncompliant@+1
+ARG ACCESS_TOKEN=${ACCESS_TOKEN}
+#   ^^^^^^^^^^^^
+
+ARG ACCESS_TOKEN=${RANDOM}
+
+ARG CCC
+ARG ${CCC}=BBB
+
+ARG ACCESS_TOKEN=""
```

Comment:
From the ticket description, this apparently should be NonCompliant.
Also an extra usecase : `ARG ACCESS_TOKEN`

Reply:
Okay. Nice catch.